paths/asset.yaml

```diff
@@ -1,16 +1,56 @@
 delete:
   description: Remove an asset
   parameters:
-    - $ref: '../parameters/X-User-Token.yaml'
     - name: id
-      required: true
       in: path
+      description: ID of the asset
+      required: true
+      schema:
+        type: integer
```

Comment:
this is correct, but I think we should change that sometime in the future to use the uuid

Reply:
Agree, same with static pages and we already have a ticket for that